src/setup_jest.ts

```diff
@@ -0,0 +1,14 @@
+const originalConsoleError = console.error
+
+// Look into removing this on the next React update to 0.16.
+console.error = (message?: any, ...optionalParams: any[]) => {
+  if (
+    typeof message === "string" &&
+    (message.includes("Accessing PropTypes via the main React package") ||
+      message.includes("React.createClass is no longer supported."))
+  ) {
+    // NOOP
+  } else {
+    originalConsoleError(message)
```

Comment:
I'm sure, yep, we don't update React / React Native often and it's tough to look at test results when every test has lines of deprecation warnings. Happy to switch to this version though 👍